src/App.css

```diff
@@ -1,39 +1,87 @@
-.App {
-  text-align: center;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  -webkit-user-drag: none;
+  -webkit-user-select: none;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+body {
+  background-color: burlywood;
+  display: grid;
+  place-content: center;
+  font-size: 2rem;
+  height: 100vh;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.container {
+  height: 75vh;
+  width: 50vh;
+  background-color: black;
+  border-radius: 1rem;
+  padding: 1rem;
+  -webkit-box-shadow: 7px 7px 5px 0 rgba(50, 50, 50, 0.75);
+  -moz-box-shadow: 7px 7px 5px 0 rgba(50, 50, 50, 0.75);
+  box-shadow: 7px 7px 5px 0 rgba(50, 50, 50, 0.75);
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+.wrapper {
+  display: grid;
+  height: 100%;
+  width: 100%;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(6, 1fr);
+  gap: 10px;
+}
+
+.screen {
+  grid-column: 1/-1;
   color: white;
+  text-align: center;
+  padding: 0.5rem;
+  font-size: 4rem;
+  overflow: hidden;
+  background-color: rgba(43, 40, 40, 0.993);
+  border-radius: 4px;
 }
```

Comment:
- [ ] Kindly style your calculator app to match the original design given. I would say your design looks nice but let's try to follow project requirements as much as we can. Please modify your styles 👍🏽


**Your Design**

<img width="584" alt="Screenshot 2023-06-06 at 11 40 14 AM" src="https://github.com/rahimialijan/math-magicians/assets/50941074/a72a5d5e-8fa2-4f7d-a094-d96d5f7edc4b">


**The Original Design**

<img width="313" alt="Screenshot 2023-06-06 at 11 40 35 AM" src="https://github.com/rahimialijan/math-magicians/assets/50941074/d68174cf-e31d-44e2-8517-0df772facb5d">